cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts

```diff
@@ -201,29 +219,36 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
    * Istio requires Prometheus operator to be installed, see previous steps.
    */
   describe('Istio', () => {
-    const istioVersion = '103.0.0%2Bup1.18.2';
-    const chartsIstioPage = `${ chartsPageUrl }&chart=rancher-istio&version=${ istioVersion }`;
-
-    const chartsPage: ChartsPage = new ChartsPage(chartsIstioPage);
-
     beforeEach(() => {
       cy.login();
+      HomePagePo.goTo();
+
+      cy.getRancherResource('v1', 'catalog.cattle.io.clusterrepos', 'rancher-charts?link=index', 200).then((resp: Cypress.Response<any>) => {
+        const version = resp.body.entries['rancher-istio'][0]['version'];
```

Comment:
Same for this one, but in reverse. There may be times in the future the first chart isn't shown so won't match up with the first in the response.